README.md

```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
 
 ### Prepare Database
 
-Run `make dev-prepare-database`
+   ```
+   Run `make dev-prepare-database`
```

Comment:
That isn't a valid command - you need to remove the single quotes, and "Run" isn't a command.
```suggestion
make dev-prepare-database
```